### DISCUSSION TITLE
Display habit plane of a grain boundary along with corresponding plane in the crystal shapes of two grains.

### DISCUSSION THREAD

**Discussion by Geordie611** [mtex-toolbox/mtex#971] [Ask Anything]
I have been working to distinguish the orientation relationship (OR) of a pair of intermetallic phases within a eutectic system through IPF density plots produced from association on a 3D data set and pole figures from a 2D EBSD data set. We believe we have correctly identified both the OR as well as the habit planes of their specific boundaries. For a sanity check, I would like to plot the habit plane as well as the specific plane in the crystal shapes of each intermetallic. However, I can't seem to find a way to do this effectively within MTEX. More specifically, I can't find a way to plot the habit plane of the grain boundary. I've read through the documentation on twinning analysis, shown the parallel planes from one grain to the other, and plotted the two crystal shapes with these and identifying how parallel the planes look. 

I've attached a drawing of what I am trying to achieve.
![Habit Plane Plot](https://user-images.githubusercontent.com/45052642/115251819-e8066980-a0f8-11eb-90e2-66dcf55460de.png)

Any help would be greatly appreciated!

**Comment by kilir** [comment]
Hi,

plotting internal planes is of now not supported (#504).
A workaround for the moment would be to define a shape based on the plane you'd like to plot, index the plane by Miller from the shape and plot it in the center of your shape (see # 504 again).

Cheers,
Rüdiger

**Reply by Geordie611** [reply]
Hi,

So I have already plotted the internal planes (using the work around described in that very discussion thread). I however was more asking about determining the plane of the GB (the black line in the schematic). The only way I've seen to compute it seems individually through the method described in the Twinning analysis documentation.

Thanks,
-Geordie

**Reply by kilir** [reply]
From a 2D EBSD map, you see the trace of the plane, but you can't determine the boundary normal direction unless you do e.g. 3D EBSD you mentioned in your original post.

Cheers,
Rüdiger
